x/auth/tx/service_test.go

```diff
@@ -565,6 +567,110 @@ func (s *IntegrationTestSuite) TestSimMultiSigTx() {
 	s.Require().Greater(res.GasInfo.GasUsed, uint64(0))
 }
 
+// Tests that all msg events included in an authz MsgExec tx
+// Ref: https://github.com/cosmos/cosmos-sdk/issues/9501
+func (s *IntegrationTestSuite) TestSim_MsgExecEvents() {
```

Comment:
I think this test should be in `x/authz`

Reply:
[Tests moved to x/authz](https://github.com/cosmos/cosmos-sdk/pull/9522/files#diff-67998452d8557f2c3f6460e17097345301363eeda37cae4e14bd7a5bb453c1aeR199-R247)